esp-radio/src/wifi/mod.rs

```diff
@@ -899,7 +899,7 @@ pub enum Capability {
 #[cfg_attr(feature = "defmt", derive(defmt::Format))]
 #[cfg_attr(feature = "serde", derive(Deserialize, Serialize))]
 #[non_exhaustive]
-pub enum Config {
+pub enum WifiApStaConfig {
```

Comment:
IMO we can drop the `Wifi` prefix, we're in the `wifi` module. But then `ApStaConfig` would be somewhat misleading as it's not a config for the ApSta mode only. Can we call this thing `ModeConfig` or something that hints at this being related to all operating modes, but also something that is not too long?